river/Touch.zig

```diff
@@ -102,6 +101,15 @@ fn handleTouchDown(
     self.wlr_cursor.absoluteToLayoutCoords(event.device, event.x, event.y, &lx, &ly);
 
     if (Cursor.surfaceAtPosition(lx, ly)) |result| {
+        switch (result.parent) {
+            .view => |view| {
+                self.seat.focusOutput(view.output);
```

Comment:
`self.seat.focus(view)` will already focus the output the view is on if it is unfocused, so you do not need to do it manually here.

Reply:
You probably copied from `checkFocusFollowCursor()`, right? If you like, you can add a commit removing it there as well.

Reply:
I did copy it from there and I did also notice that `seat.focus` sets the output. I'll fix it in both places.